src/style.css

```diff
@@ -1,3 +1,92 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  --header-height: 60px;
+  background-color: rgb(209, 207, 207);
+}
+
+.page__body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  height: var(--header-height);
+
+  display: flex;
+  justify-content: flex-end;
+  background-color: white;
+  align-items: center;
+  padding: 0 50px;
+  cursor: pointer;
+}
+
+.logo {
+  margin-right: auto;
```

Comment:
you just need to set justify-content: space-between; for .header instead this
![image](https://github.com/mate-academy/layout_moyo-header/assets/96265844/3a6dc71f-4153-4612-b9bd-15a14ffb0327)